### DISCUSSION TITLE
Failed to create python3 virtual environment in lsp and mason

### DISCUSSION THREAD

**Discussion by tonmoy998** [LazyVim/LazyVim#1267] [Q&A] [answered]
![Screenshot_20230801_104534](https://github.com/LazyVim/LazyVim/assets/74363531/3450459f-c76b-4e10-a1e0-f20e2c19cbce)

**Comment by jyuan0** [comment]
Does `:checkhealth mason` show any errors?

**Reply by tonmoy998** [reply]
I guess not
![Screenshot_20230801_121814](https://github.com/LazyVim/LazyVim/assets/74363531/30ea2135-2882-4b3c-821c-787ca397cf59)

![Screenshot_20230801_121820](https://github.com/LazyVim/LazyVim/assets/74363531/6a5816a9-7a9d-4129-8fc5-7b3abc79a088)

**Reply by dpetka2001** [reply, marked as answer]
Can you try installing `sudo apt-get install python3-venv`?

**Reply by tonmoy998** [reply]
Thank you mate ☺️

**Comment by jyuan0** [comment]
Can you get more info from the errors?

https://www.reddit.com/r/neovim/comments/100srii/comment/j2jwdhy/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button

**Reply by tonmoy998** [reply]
Thanks for your suggestions.
